### PR TITLE
added support for URL image in ImageContainer2D

### PR DESCRIPTION
**Pull request by antrikshmisri**
This adds support for fetching the image from a URL. The image is stored in the directory temporarily until the image has been processed, then it is removed from the directory. I'll try to look at other approaches that may be more flexible than this in the meantime and update this if necessary.

### CODE REVIEW

**Review by skoudoro — COMMENTED** [review]
Hi @antrikshmisri,

Thank you for this. 

I recommend you:
- to remove the parameter `is_url` everywhere.
- To check if the filename starts with `http://` or `https://`
- Be aware of your `os.remove` because it seems to do it every time.

So in your `load_image` function you can do: `is_url = filename.startswith("http://") or filename.startswith("https://")`

Thank you

File: fury/io.py [hunk 1]
```diff
@@ -24,6 +25,13 @@ def load_image(filename, as_vtktype=False, use_pillow=True):
         desired image array
 
     """
+    is_url = filename.startswith('http://') or filename.startswith('https://')
```
Comment (skoudoro) [inline fury/io.py:28]: `filename.lower().startswith('http://')` in case you have `HTTP://`

**Review by skoudoro — COMMENTED** [review]
Thank you for the update.

Please, Can you add a test @antrikshmisri?

After that it is ready to be merged

File: fury/io.py (hunk 1 above)
Comment (antrikshmisri) [reply fury/io.py]: Added the tests/requested changes.

**Review by skoudoro — APPROVED** [review]
Looks good to me.

Can you have a quick look @Nibba2018? 

I will merge this PR by the end of the day

**Comment by skoudoro** [comment]
Thank you @antrikshmisri! merging

**Pull request merged**